grappelli/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -179,10 +182,7 @@
                 updateRelatedObjectLinks(this);
             }
         });
-        // GRAPPELLI CUSTOM
-        /* triggering select means that update_lookup is triggered with
-        generic autocompleted (which would empty the field) */
```

Comment:
apparently it does not empty it.

Reply:
With my setup it does (with generic autocompletes).